app/model/bank_account.py

```diff
@@ -41,12 +43,9 @@ class BranchDetails(db.Model):
     """Create table BranchDetails in the database"""
 
     __tablename__ = 'BranchDetails'
-    id = db.Column(db.Integer, primary_key=True)
-    branch_code = db.Column(db.Integer, nullable=False)
+    branch_id = db.Column(db.Integer, primary_key=True)
```

Comment:
instead of `branch_id` use `id` inside `BranchDetails` table and when you refer it in some other table, call it `branch_id`